vscode-smart-contracts/src/extension.ts

```diff
@@ -125,7 +126,7 @@ const taskProvider: vscode.TaskProvider = {
       const workspaceFolder = vscode.workspace.getWorkspaceFolder(
         vscode.Uri.file(cwd)
       );
-      const resolvedTask = await cargoConcordium.build(
+      const resolvedTask = await cargoConcordium.test(
```

Comment:
Will this run the tests using `cargo concordium test`? Or just `cargo test`?

Reply:
It will use `cargo concordium test`
The extension does not provide anything for running `cargo test`

Reply:
Ok, great!